config/Makefile.mingw

```diff
@@ -91,6 +91,7 @@ FLAMBDA=false
 WITH_FLAMBDA_INVARIANTS=false
 WITH_SPACETIME=false
 ENABLE_CALL_COUNTS=false
+WITH_STATMEMPROF=false
```

Comment:
These four files will shortly be removed, so please feel free to save your time and not update them!

Reply:
Nice! Is this the case for all the `Makefile.xxxx` targetting the various Windows platforms?

Reply:
Indeed - all the Windows ports are now using `configure`